android/app/build.gradle

```diff
@@ -234,7 +234,7 @@ task hemroidBuild(type: Exec) {
     def rootDir = project.rootDir
     def localProperties = new File(rootDir, "local.properties")
 
-    def ndkDir = "$System.env.ANDROID_NDK_HOME"
+    def ndkDir = "/Users/cappuccino/Library/Android/sdk/ndk-bundle"
```

Comment:
This probably should not be there.

Reply:
Correct, these are a sign of my battles with getting a dev build 😄